### PR TITLE
Add chat bubble width param

### PR DESCRIPTION
**Pull request by dawoodkhan82**
## Description
Adds the highly requested param to change the chatbot message bubble width to fit message content.

Screenshot 2023-08-24 at 3 37 09 PM

Closes: #4420

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/components/chatbot.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def __init__(
         show_share_button: bool | None = None,
         show_copy_button: bool = False,
         avatar_images: tuple[str | Path | None, str | Path | None] | None = None,
+        chat_bubble_fit_content: bool = False,
```
Comment (abidlabs) [inline gradio/components/chatbot.py:58]: I'm wondering if we can shorten the name of the parameter? Maybe `bubble_full_width` (which is default `True`)

**Comment by abidlabs** [comment]
LGTM @dawoodkhan82! Could we add a story for this as well?

**Comment by austinmw** [comment]
Does this work for `gr.ChatInterface`?

**Comment by abidlabs** [comment]
Not necessary, you can pass in a custom `Chatbot` instance as the `chatbot` parameter to `gr.ChatInterface` to get this behavior. 

See this Space by @yvrjsharma for example: https://huggingface.co/spaces/ysharma/falcon-180b-demo/blob/92739960e894d417f240b2ec6f55e92ab6b0c788/app.py#L124

**Comment by lucasjinreal** [comment]
@abidlabs  thanks for this useful feature.

Is the possible add further a max_width here when full_width is false?

**Pull request merged**